source/solvers/heat_transfer_assemblers.cc

```diff
@@ -209,6 +209,7 @@ HeatTransferAssemblerBDF<dim>::assemble_matrix(
     scratch_data.grad_specific_heat_temperature;
 
   // Copy data elements
+  // ?? do we need strong residual here??
```

Comment:
I don't know if you mean't to keep this comment or if you want to delete it

Reply:
I kept these comments to ask them from Bruno. I will remove them after that.